code-gurus/nisisi-backend
# Add Security Firewall

# PR Details


- Will add security question firewall 

## How Has This Been Tested




- After cloning the repository, make sure that you've created a database with mongo

- Then, make sure that the server is running, if not run it with the following command `npm run dev`

- Open postman and send a request to the backend with the following url `localhost:3000/api/users/questions`,
- Check your email and you will see the template and link 

## Types of changes




## Checklist





## Screenshots
